src/main/resources/application.properties

```diff
@@ -42,7 +42,7 @@ alpine.database.mode=external
 alpine.database.port=9092
 # Required
 # Specifies the JDBC URL to use when connecting to the database.
-#alpine.database.url=jdbc:postgresql://localhost:5432/dtrack
+alpine.database.url=jdbc:postgresql://localhost:5432/dtrack
```

Comment:
unintentional commit maybe. We have intellij profile that sets these properties for local run